tmeister/static/src/toggle-meister.js

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
 import ReactDom from 'react-dom';
+// config dayjs
+import dayjs from 'dayjs'
+import calendar from 'dayjs/plugin/calendar'
+dayjs.extend(calendar)
```

Comment:
To support the `calendar` api from moment I need this opt-in behavior/plugin

Reply:
I don't see usage of `dayjs` here...